site/en/setup.md

```diff
@@ -14,48 +14,57 @@ to achieve the workflow best suited for OCRing your content.
+  - Other Linux distributions or Ubuntu versions can also be used, though some instructions have to be adapted (e.g. package management, locations of some files)
+  - With Windows Subsystem for Linux (WSL), a feature of Windows 10, it is [also possible to set up an Ubuntu 18.04 installation within Microsoft Windows](https://github.com/OCR-D/ocrd-website/wiki/OCR-D-on-Windows)
+  - OCR-D can be deployed on an [Apple MacOSX machine using Homebrew](https://github.com/OCR-D/ocrd-website/wiki/OCR-D-on-macOS)
+  
+</details>
+<details>
+<summary>- Python 3.5, 3.6 or 3.7</summary>
+  
+  - OCR-D's target Python version is currently Python 3.5 which we will continue to support until at least the end of 2020
+  - Python 3.6 and 3.7 are also tested and supported
+  - We currently **cannot fully support Python 3.8**, because there currently (July 2020) are no pre-built Python packages for Tensorflow and other GPU related software). We expect to unconditionally support Python 3.8 in Q1 2021 the latest.
+  
+</details>
 For installation on Windows 10 (WSL) and macOS see the setup guides in the [OCR-D-Wiki](https://github.com/OCR-D/ocrd-website/wiki)
 
 Alternatively, you can use [Docker](https://hub.docker.com/u/ocrd). This way, you will only have to meet the minimum requirements for 
 free disk space. But you can use any operating system you want and do not have to worry about the Python version. 
 
 ## Installation
 
-All [OCR-D modules](https://github.com/topics/ocr-d) follow the same
-[interface](https://ocr-d.github.io/cli) and common design patterns. So once
-you understand how to install and use one project, you know how to install and
-use all of them.
-
-There are four ways to install OCR-D modules:
+There are two ways to install OCR-D modules:
 
   1. [Using](#ocrd_all-via-docker) the [ocrd_all prebuilt Docker images `ocrd/all`](https://hub.docker.com/r/ocrd/all) to install a module collection (**recommended**)
   2. [Using](#ocrd_all-natively) the [ocrd_all git repository](https://github.com/OCR-D/ocrd_all) to install selected modules natively
-  3. [Installing](#individual-installation) modules individually, either via Docker or natively (not recommended)
-  4. [Using](#installation-of-ocrd_framework) the [ocrd_framework repository](https://github.com/VolkerHartmann/ocrd_framework) to install all modules, Taverna workflow and research data repository as a local Docker image
 
 We recommend using the prebuilt Docker images since this does not require any changes to
 the host system besides [installing Docker](https://hub.docker.com/r/ocrd/all).
 
-We do not recommend installing modules individually, because it can be difficult to catch all dependencies, 
+For developers it might be useful to [install the modules individually](#individual-installation), either via Docker or natively.
```

Comment:
Should we even mention this here? We could move that section to the developer's guide (which is woefully out-of-date as well 😨)

Reply:
I don't really mind. Would make the setup guide even clearer if we could just remove it completely.

Reply:
IMHO we should keep mentioning this option in the setup guide (both as a subsection and a short mention of it above). There are still (to this day) users who want maximal control and don't trust (or want to learn about) ocrd_all's makefile. From their perspective, it's the lowest threshold for entering OCR-D.

Reply:
I think it is okay to reference this here as long as we restrict any details to another guide/doc.